src/components/benefit-cta-link.vue

```diff
@@ -63,11 +63,17 @@ export default {
     },
   },
   methods: {
+    hasFinishedSimulation() {
+      return !this.store.calculs.resultats._id
```

Comment:
Ah du coup c'est l'inverse `this.store.calculs.resultats._id` sans le `!`, et ligne 71 on renvoi seulement si `!this.hasFinishedSimulation()`

Reply:
Yes, merci